CMakeLists.txt

```diff
@@ -1150,6 +1150,22 @@ int main() {
   endif()
 endif()
 
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
+  include(CheckCSourceCompiles)
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -march=armv8-a+sve")
+  check_c_source_compiles("#include <arm_sve.h>
+int main() {
+  svfloat64_t a;
+  a = svdup_n_f64(0);
+  return 0;
+}" COMPILER_HAS_ARM_SVE)
+
+  if(COMPILER_HAS_ARM_SVE)
+    string(APPEND CMAKE_CXX_FLAGS " -DCOMPILER_HAS_ARM_SVE")
+  endif()
+  set(CMAKE_C_FLAGS ${ORIGINAL_CMAKE_C_FLAGS})
+endif()
+
```

Comment:
This is a compile time check and is somewhat irrelevant during the compile, isn't it?

Reply:
I have made the changes and pushed in the main SVE PR.
https://github.com/pytorch/pytorch/pull/119571

**Here's the exact commit with the change:**
https://github.com/pytorch/pytorch/pull/119571/commits/47dcc0229a2b6fb1638daee381a44d6d2a0a73b9